src/pybind/mgr/volumes/fs/vol_spec.py

```diff
@@ -11,6 +16,8 @@ class VolSpec(object):
     DEFAULT_NS_PREFIX = "fsvolumens_"
     # default mode for subvol prefix and group
     DEFAULT_MODE = 0o755
+    # internal directories
+    INTERNAL_DIRS = [Group.NO_GROUP_NAME, Index.GROUP_NAME, Trash.GROUP_NAME, SubvolumeBase.LEGACY_CONF_DIR]
```

Comment:
I don't see this in `main` PR ? Is this part of another PR which is not backported ?

Reply:
This depends on https://github.com/ceph/ceph/pull/47460 which is qa passed. Wait for this to be merged and please rebase after that.

Reply:
If https://github.com/ceph/ceph/pull/47460 gets merged before this PR then I will rebase my PR.
If https://github.com/ceph/ceph/pull/47460 is not going to merge before my PR then above changes are needed as my code depends on https://github.com/ceph/ceph/pull/47460